ner/routers/ner.py

```diff
@@ -8,6 +8,7 @@
 
 @router.post('/extract_name')
 def extract_name(input: dict):
-    ner = NER()
+    ner = NER(input['id'])
```

Comment:
We can use UUID to generate an ID from a Python method and pass it to the front end.

Reply:
done